multipool-tracker/Cargo.toml

```diff
@@ -29,3 +29,7 @@ colored = "2.1.0"
 
 env_logger = "0.10.2"
 log = "0.4.20"
+
+[[test]]
```

Comment:
Why this was added to toml? Tests in 
```rust
#[cfg(test)]
```
Work perfectly without adding custom test specification for cargo